content/concept-xrp-ledger-intro.md

```diff
@@ -0,0 +1,88 @@
+
+- A famously strict and thorough code review process
+- Comprehensive code coverage and unit tests
+- Regularly running automated checks for potential vulnerabilities and memory leaks
+- Regularly commissioning external reviews by professional organizations
+
+The XRP Ledger also has a built-in "Amendments" system for evolving the existing feature set and transitioning from old rules to new ones. Rather than fracturing the ecosystem or causing uncertainty whenever a transition occurs, any changes to transaction processing in the XRP Ledger are switched on by Amendments that require sustained approval from validators over 2 weeks, giving everyone time to evaluate the changes before they apply, and upgrade their servers appropriately. The system also prevents outdated servers from reporting incorrect data when the rules of the network have changed. For more on how Amendments work, see [Amendments](concept-amendments.html).
+
+
+## Secure, Adaptable Cryptography
+[Secure, Adaptable Cryptography]: #secure-adaptable-cryptography
+
+Cryptography is one of the hardest parts of any distributed system, and a mistake can lead to money stolen by malicious actors anywhere in the world. The XRP Ledger uses industry-standard schemes for signing and verifying transactions, algorithms that have successfully protected hundreds of billions of US dollars' worth of value for many years. The XRP Ledger also layers multi-signing functionality so you can use multi-factor authorization or split keys across multiple people as a backup, and provides new algorithms with a path to migrate the keys you use if a breakthrough in cryptography makes the old algorithms obsolete.
+
+For more information, see [Cryptographic Keys](cryptographic-keys.html) and [Multi-signing](reference-transaction-format.html#multi-signing).
+
+
+## Smart Features for Smart Contracts
+[Smart Features for Smart Contracts]: #smart-features-for-smart-contracts
+
+Besides simple value transfer via XRP payments, the XRP Ledger has several advanced features that provide useful functions for building applications that use Internet of Value to serve previously unknown or impractical needs. Rather than running applications as "smart contracts" in the network itself, the XRP Ledger provides tools for contracts, while letting the applications themselves run anywhere, in whatever environment or container is appropriate. This "keep it simple" approach is flexible, scalable, and powerful.
+
+A sample of advanced features in the XRP Ledger:
+
+- [Payment Channels](tutorial-paychan.html) allow asynchronous balance changes as fast as you can create and validate signatures.
+- [Escrow](concept-escrow.html) locks up XRP until a dedicated time or cryptographic condition.
```

Comment:
Consider changing "until a dedicated time or cryptographic condition." to "until a dedicated time or a cryptographic condition is met."